app/src/routes/event/[id]/+page.svelte

```diff
@@ -24,9 +24,11 @@
     </a>
   </div>
   <EventSummary {event} />
-  {#if auth?.user}
-    <EventFormInternal {event} {data} />
-  {:else}
-    <EventFormExternal {event} {data} />
+  {#if event.openForExternals}
```

Comment:
Tenker det er OK å ikke vise noe her dersom man ikke har tilgang

Reply:
Ja, tenkte det samme 👌

Reply:
Men dette ble feil sted å ha det? Nå vises ikke påmeldingen for interne heller

Reply:
Oops ja, bra sett og fikset!